rainier-core/src/main/scala/com/stripe/rainier/core/RVG.scala

```diff
@@ -0,0 +1,21 @@
+package com.stripe.rainier.core
+
+case class RVG[T](value: RandomVariable[Generator[T]]) {
```

Comment:
Probably worth a quick comment explaining what this is.

Reply:
Agreed but will leave for a follow up PR.